src/views.tsx

```diff
@@ -76,7 +76,11 @@ export class TasksTimelineView extends BaseTasksView {
         const adapter = new ObsidianTaskAdapter(this.app);
         adapter.generateTasksList(fileIncludeFilter, fileExcludeFilter, fileIncludeTagsFilter, fileExcludeTagsFilter)
             .then(() => {
-                const taskList = adapter.getTaskList();
+				let taskList = adapter.getTaskList();
+
+				const sortOptionFn = (0, eval)(this.userOptionModel.get("sort") || "")
+				if (sortOptionFn) taskList = taskList.sort(sortOptionFn)
```

Comment:
sorting tasks here might not work cauze some fields of the task items might not get parsed yet.
you need to sort after all the meta infos are parsed.